lib/domain/repository/chat.dart

```diff
@@ -388,3 +397,14 @@ class RxChatMember implements Comparable<RxChatMember> {
     return result;
   }
 }
+
+/// A scroll position anchored to a specific chat item and its offset.
+class ChatScrollPosition {
```

Comment:
Did you take a look at contribution guide after the last review request to do so?

https://github.com/team113/messenger/blob/main/CONTRIBUTING.md#documentation-1

> Documentation must follow [Effective Dart](https://dart.dev/guides/language/effective-dart) official recommendations with the following exception:
>
> - prefer omitting leading A, An or The article.